meerk40t/core/elements.py

```diff
@@ -311,6 +311,20 @@ def plugin(kernel, lifecycle=None):
             },
         ]
         kernel.register_choices("preferences", choices)
+        choices = [
+            {
+                "attr": "classify_debug",
+                "object": elements,
+                "default": False,
+                "type": bool,
+                "label": _("Debug Classification"),
+                "tip": _("Shows info about the classification process in the console"),
+                "page": "Classification",
+                "section": "_AA_Debug",
+                "hidden": True,
```

Comment:
We're now giving ourselves fancy special options in preferences so if you have developer mode on you have the convenience of pressing this button rather than doing something like `batch add channel print classification`